README.md

```diff
@@ -1,60 +1,65 @@
-   in the project's repository to the _Exceptions_ section in the [TERMS](TERMS.md).
-  - If no exceptions are needed, remove that section from TERMS.
-5. If working with an existing code base, answer the questions on the [open source checklist](opensource-checklist.md)
-6. Delete these instructions and everything up to the _Project Title_ from the README.
-7. Write some great software and tell people about it.
+Automating CFPB's [Open Source Checklist](https://github.com/cfpb/open-source-project-template/blob/master/opensource-checklist.md) for easy release of internal source code.
 
-> Keep the README fresh! It's the first thing people see and will make the initial impression.
+![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
 
-----
 
-# Project Title
+## Dependencies
 
-**Description**:  Put a meaningful, short, plain-language description of what
-this project is trying to accomplish and why it matters.
-Describe the problem(s) this project solves.
-Describe how this software can improve the lives of its audience.
+- [Node.js](https://nodejs.org/en/)
 
-Other things to include:
+## Installation
 
-  - **Technology stack**: Indicate the technological nature of the software, including primary programming language(s) and whether the software is intended as standalone or as a module in a framework or other ecosystem.
-  - **Status**:  Alpha, Beta, 1.1, etc. It's OK to write a sentence, too. The goal is to let interested people know where this project is at. This is also a good place to link to the [CHANGELOG](CHANGELOG.md).
-  - **Links to production or demo instances**
-  - Describe what sets this apart from related-projects. Linking to another doc or page is OK if this can't be expressed in a sentence or two.
+To use [scrub.sh](scrub.sh) to scrub GHE urls:
 
+### Add environment variable
 
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
+1. Add `.env` and `scrub.sh` to your project's `.gitignore` list.
+1. Copy [`.env_SAMPLE`](.env_SAMPLE) to your project's root.
+1. Rename `.env_SAMPLE` to `.env`.
+1. Edit `fake.ghe.url` to point to the real GHE url (do not include `https://`).
+1. `cd` into your project repo root to activate the new environment variable. You should receive a warning and then a confirmation message: 
 
-![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
+![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot-env.png)
 
+## Usage
 
-## Dependencies
+1. In Terminal, from project root, run:
 
-Describe any dependencies that must be installed for this software to work.
-This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
-If specific versions of other software are required, or known not to work, call that out.
+  ```bash
+sh ./scrub.sh
```

Comment:
No need to include `sh ` here -- the [shebang](https://github.com/cfpb/open-source-checklist/pull/1/files#diff-53423e48e1aa03d9b56b3abab8416cb9R1) specifies that the file should be passed to `sh`.

Instead, make the script executable by doing `chmod +x scrub.sh`. You can then execute it with just `./scrub.sh`.